Cargo.toml

```diff
@@ -21,26 +22,35 @@ byteorder = "1.5.0"
 clap = { version = "4.5.18", features = ["derive"] }
 cusf_sidechain_types = { git = "https://github.com/LayerTwo-Labs/cusf_sidechain_types" }
 ed25519-dalek-bip32 = "0.3.0"
-env_logger = "0.11.5"
```

Comment:
I'll admit I'm no Rust expert - but I really like env_logger. It seems like _a lot_ of Rust apps and libraries configure logging through `RUST_LOG`, notably that's how BDK does it. So by setting RUST_LOG it would be possible to our logs, as well as logs of our dependencies. Just my two cents!